vic/drivers/shared_image/src/vic_store.c

```diff
@@ -25,12 +25,14 @@
  *****************************************************************************/
 
 #include <vic_driver_shared_image.h>
+#include <vic_driver_cesm.h>
```

Comment:
Yep - I'm doing this now in the `vic_cesm_run` function. Take a look at the way I'm reassigning `filenames.statefile` though - it didn't work the way you suggested (I don't know why, since I'm new to this) and this was a way that worked.